### DISCUSSION TITLE
Using configs from different config groups

### DISCUSSION THREAD

**Discussion by dorotat-nv** [ory/hydra#3574] [Q&A]
Hi! In my use cases, we have some dependencies between hydra configs for training different models - different config groups. 

However, the configs may be located in different directories. I am wondering how to tackle it without the limitation of `hydra.searchpath` being allowed only in the primary config. Ie an example

`config /A/conf/A.yaml `- some base config
`config /B/conf/B.yaml` that uses as a default `/A/conf/A.yaml `with `hydra.search.path = "/A/conf"`
`config /B/conf/B1.yaml` that uses as a default `/B/conf/B.yaml` from the same folder
`config /C/conf/C.yaml` that uses as a default `/B/conf/B.yaml` with `hydra.search.path = "/B/conf"`
Hydra will process with no problems configs in (1) and (2) but throws errors at (3) and (4) since they contain inherited `hydra.searchpath` from the defaults configurations. 

Ideally, I would like to come up with a solution that attaches `hydra.searchpath`s from the non-primary configs to the `hydra.searchpath` as this approach gives the most visibility to where the configs are located

**Comment by vinckr** [comment]
Hey, that is an interesting use case. 
Have you checked out the approach that Ory Network uses with uploading the config via CLI/CI - would something like that work for you?
